Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -159,6 +159,7 @@ namespace AZ::Reflection
                 bool m_skipLabel = false;
                 AZStd::string m_labelOverride;
                 bool m_disableEditor = false;
+                bool m_disableContainer = false;
```

Comment:
This name doesn't really make sense. It's not really specific to containers, but to nodes with the ReadOnly attribute. Was trying to avoid using ReadOnly since we could just register the ReadOnly attribute in that case.

Reply:
Talked with Alex again and made some sense of things. At first I was using the same attribute to determine whether containers and elements should have their action buttons disabled, but that was incorrect and causing confusion on my part.

If there is a better way to do this @NicholasVanSickle then please let me know.